README.md

```diff
@@ -1,2 +1,16 @@
-# kf-sphinx-docs-theme
-💄 Kids First Sphinx Docs Theme
+# Kids First Sphinx Docs Theme
+
+A Kids First styled Sphinx theme based on https://github.com/rtfd/sphinx_rtd_theme
+
+Sphinx is a web documentation generator for Python applications
+
+## Installation
+```
+pip install -e git+https://github.com/kids-first/kf-sphinx-docs-theme.git#egg=kf-sphinx-docs-theme
+```
+
+## Test
+```
+pip install -r dev-requirements.txt
+python -m pytest tests
+```
```

Comment:
add a "how to use" section plz

Reply:
ah yes 👍

Reply:
ok, now how to include this in another project in a way that won't immediately fall behind this repo?